conans/client/conf/config_installer.py

```diff
@@ -0,0 +1,116 @@
+import os
+from conans.tools import unzip
+import shutil
+from conans.util.files import rmdir, load, mkdir
+from conans.client.remote_registry import RemoteRegistry, Remote
+from conans import tools
+from conans.errors import ConanException
+
+
+def _handle_remotes(registry_path, remote_file, output):
```

Comment:
should do the same as remote_registry._parse?

Reply:
This is related to my question, check above. I was thinking about using the same syntax of the ``conan remote list``, not the syntax of the registry file.

Reply:
I would go for the `registry.txt` syntax, two files with different formats being the same is not good. So probably the code of _handle_remotes can be refactored with the "parse" from the registry.